testpar/t_select_io_dset.c

```diff
@@ -1588,7 +1588,7 @@ test_multi_dsets_no_bkg(hid_t fid, unsigned chunked, unsigned dtrans, unsigned s
                  mwbuf ? "mwbuf" : "nomwbuf");
 
         /* Flip a coin to see if we're doing type conversion */
-        tconv = rand() % 2;
```

Comment:
It's surprising to me that this test doesn't call srand (neither does the serial version). We should probably change that, though it doesn't necessarily need to be in this PR.

Reply:
I also thought that was weird, but the docs say not calling srand is like using 1 as the seed, IIRC, and since we just use 42 as the seed in the parallel test I wasn't going to worry too much about it.